Directory.Build.props

```diff
@@ -4,4 +4,13 @@
     <__XFBuildTasksLocation Condition="'$(__XFBuildTasksLocation)' == '' AND '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory).nuspec\netstandard2.0\</__XFBuildTasksLocation>
     <__XFBuildTasksLocation Condition="'$(__XFBuildTasksLocation)' == ''">$(MSBuildThisFileDirectory).nuspec\net46\</__XFBuildTasksLocation>
   </PropertyGroup>
- </Project> 
+  <PropertyGroup>
+    <Company>Microsoft</Company>
+    <Product>Xamarin.Forms</Product>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(CI)' == ''">
```

Comment:
this might be nice to have, but not necessary as the build is still changing

Reply:
Is nice to have and follows what others are doing in the company, I see no reason we can't add this. CI will always be changing tbh.